src/styles/Button.module.scss

```diff
@@ -1,42 +1,49 @@
 @import './Constants.scss';
 
+@mixin disableDefaultButtonStyles {
+  font-family: inherit;
+  background: none;
+  border: none;
+  padding: 0;
+}
+
+$hover-transition-time: 0.2s;
+$click-transition-time: 0.01s;
+$button-height: 3.25rem;
+
 .background {
   background-color: black;
 }
 
 .button {
-  background: none;
-	border: none;
-	padding: 0;
-	outline: inherit;
-  display: inline-block;
-  margin: 1rem 1rem;
-  box-sizing: border-box;
+  @include disableDefaultButtonStyles;
```

Comment:
Te kilka zagadkowych właściwości wyrzuciłem do mixina, którego nazwa dużo więcej mówi o tym, co te właściwości robią. Takiego mixina załączam sobie @include'm

Reply:
Oprócz tego, zauważ też to:
- display: inline-block jest bezużyteczny, bo button defaultowo ma inline-block
- margin: 1rem 1rem jest nieprawidłowo napisane, wystarczy margin: 1rem (niżej w kodzie podobne błędy)
- box-sizing: border-box jest również defaultowym parametrem buttona, więc również do usunięcia